logger/zap.go

```diff
@@ -0,0 +1,168 @@
+package logger
+
+import (
+	"log"
+	"os"
+
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+type ZapLogger struct {
+	Zap *zap.SugaredLogger
+}
+
+var zapLogger *ZapLogger
+
+func init() {
+	logLevelEnv := os.Getenv("LOG_LEVEL")
```

Comment:
The log level must be configurable by the global config file.
c.f. https://github.com/hyperledger-labs/yui-relayer/blob/v0.4.3/cmd/config.go#L134